lm/srilm.py

```diff
@@ -410,7 +415,7 @@ def get_ppl(self):
                     if ln == "sentences,":
                         self.out_num_sentences.set(int(line[idx - 1]))
                     if ln == "words,":
-                        self.out_num_words.set(int(line[idx - 1]))
+                        self.out_num_words.set(float(line[idx - 1]))
```

Comment:
why would there ever be a non-integer number of words?

Reply:
I now have large corpora where the count switches to a scientific representation within the ppl log.

Reply:
But 
```
>>> int(1e6)
1000000
````
that is fine.. or is the number larger than the maximum int?

Reply:
yes it was quite large and the job crashed complaining about the int conversion.. I cant find the crashed job anymore.. would probably need some digging..

Reply:
There is no problem with large ints, but int can indeed not parse strings with scientific notation
```
>>> int(2**1024-1)
179769313486231590772930519078902473361797697894230657273430081157732675805500963132708477322407536021120113879871393357658789768814416622492847430639474124377767893424865485276302219601246094119453082952085005768838150682342462881473913110540827237163350510684586298239947245938479716304835356329624224137215
>>> int(1e6)
1000000
>>> int("1e6")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: '1e6'
>>> float("1e6")
1000000.0
>>> int(float("1e6"))
1000000
>>> int(eval("1e6"))
1000000
```
maybe we could do
```suggestion
                        self.out_num_words.set(int(float(line[idx - 1])))
```
instead?